packages/cli/commands/project/cloneApp.js

```diff
@@ -81,6 +82,15 @@ exports.handler = async options => {
       });
       appId = appIdResponse.appId;
     }
+    const selectedApp = await fetchPublicAppMetadata(appId, accountId);
+    // preventProjectMigrations returns true if we have not added app to allowlist config.
+    // listingInfo will only exist for marketplace apps
+    const { preventProjectMigrations, listingInfo } = selectedApp;
+    trackCommandMetadataUsage(
```

Comment:
I think we can remove this one (per my comment above). So the three tracking calls in this file will be:
- trackCommandUsage (our typical command usage tracking, without appID)
- trackCommandMetadataUsage (for successful runs)
- trackCommandMetadataUsage (for failed runs)

Reply:
Laurie asked for more information about tracking whether the apps are marketplace listed but specified that it's nice to have, not a must. Should I remove?

Reply:
Ah ok. In that case can we include the listing info in the success and failure tracking requests? Also, what's the shape of `listingInfo`? Is is a string?

Reply:
`listingInfo` is an object. Oh, I should use the spread operator then, correct?

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1127/commits/32cd50327d070cff90dc3dc3b07ec5a22bd993d8, let me know if I need to add the spread operator.